Make okonomiyake aware of Alpine to fix an KeyError exception.
diff --color -rupN a/okonomiyaki/platforms/_platform.py b/okonomiyaki/platforms/_platform.py
--- a/okonomiyaki/platforms/_platform.py	2023-06-29 19:20:58.000000000 +0200
+++ b/okonomiyaki/platforms/_platform.py	2024-07-04 12:48:42.011174306 +0200
@@ -30,6 +30,7 @@ class FamilyKind(enum.Enum):
     mac_os_x = 2
     windows = 3
     solaris = 4
+    alpine = 5
 
 
 @enum.unique
@@ -43,6 +44,7 @@ class NameKind(enum.Enum):
     windows = 5
     solaris = 6
     rocky = 7
+    alpine = 8
 
 
 NAME_KIND_TO_PRETTY_NAMES = {
@@ -54,6 +56,7 @@ NAME_KIND_TO_PRETTY_NAMES = {
     NameKind.debian: "Debian",
     NameKind.unknown: "Unknown distribution",
     NameKind.rocky: "Rocky Linux",
+    NameKind.alpine: "Alpine Linux",
 }
 
 
@@ -185,6 +188,8 @@ def _guess_platform_details(os_kind):
             family_kind = FamilyKind.rhel
         elif name_kind == NameKind.unknown and 'debian' in like:
             family_kind = FamilyKind.debian
+        elif name_kind == NameKind.alpine:
+            family_kind = FamilyKind.alpine
         else:
             raise OkonomiyakiError("Unsupported platform: {0!r}".format(name))
         return family_kind, name_kind, release
